pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -537,7 +557,7 @@ func (r *ReconcileHyperConverged) ensureCDI(instance *hcov1alpha1.HyperConverged
 	// Previous versions used to have HCO-operator (scope namespace)
 	// as the owner of CDI (scope cluster).
 	// It's not legal, so remove that.
-	if (len(existingOwners) > 0) {
+	if len(existingOwners) > 0 {
```

Comment:
this change seems unrelated to the purpose of the PR. please use another PR for code cleanup.

Reply:
Done